src/platform/user/profile/vet360/containers/AddressValidationView.jsx

```diff
@@ -0,0 +1,334 @@
+import React from 'react';
```

Comment:
This file is very similar to the existing `src/platform/user/profile/vet360/containers/AddressValidationModal.jsx` but does not use a Modal component.

Reply:
I think we can do some refactoring in the entire address validation flow. For example, I'm pretty sure we can ignore any address suggestions we get back from the validation API that are not confirmed addresses.

Reply:
will definitely take a pass at cleaning this up when solid unit/integration tests are in place